handler.py

```diff
@@ -35,7 +35,7 @@ def main(event, context):
     # set environment variables
     os.environ['BIOINDEX_S3_BUCKET'] = s3_bucket
     os.environ['BIOINDEX_RDS_SECRET'] = rds_secret
-    os.environ['BIOINDEX_RDS_SCHEMA'] = rds_schema
+    os.environ['BIOINDEX_BIO_SCHEMA'] = rds_schema
```

Comment:
Bioindex uses BIOINDEX_BIO_SCHEMA to determine db schema to use. https://github.com/broadinstitute/dig-bioindex/blob/c9b668cee6197b1a86c76ad73557d75bd597c601/bioindex/lib/config.py#L174-L177